common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -5,26 +5,23 @@
     <LangVersion>latest</LangVersion>
     <StartupObject>SmokeTest.Program</StartupObject>
   </PropertyGroup>
-
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.4.0-dev.*" />
```

Comment:
These other references that are still using dev seem like they should be updated.

Reply:
Also, you could use release version ranges in references like "4.*" to get the latest. Unfortunately, nuget still requires that prereleases use something like what you're doing (or even just "4.*-*", IIRC). Not sure if that will make things easier, though.